cymbaline/generators.py

```diff
@@ -0,0 +1,63 @@
+from cymbaline.data import DataProvider
+from cymbaline.language import LanguageToolkit
+
+import random
+
+
+class Generator:
+    def __init__(self, context_word=None, context_vector=None):
+    """Initiates Generator with either single word or vector, which defines the context"""
+        if(context_word == None and context_vector == None):
+            raise TypeError("Either `context_word` or `context_vector` argument is required.")
+
+        self._context_word = context_word
+        self._context_vector = context_vector
+
+        self.lt = LanguageToolkit(context_word=context_word, context_vector=context_vector)
+    
+    @property
+    def context_vector(self):
+        return self.context_vector
+
+    @context_vector.setter
+    def context_vector(self, value):
+        self.lt = LanguageToolkit(context_word=None, context_vector=value)
+
+    @property
+    def context_word(self):
+        return self.context_vector
+    
+    @context_word.setter
+    def context_word(self, value):
+        self.lt = LanguageToolkit(context_word=value, context_vector=None)
+
+    def generate(self):
+        raise NotImplementedError("Abstract `Generator` base class method has been called.")
+
+    def _generate_n_syllables_stich(self, n):
+        stich = ""
```

Comment:
What is the variable `stich` responsible for? The name could be a little bit more self-descriptive.

Related? https://www.youtube.com/watch?v=VbfpW0pbvaU

Reply:
Stich is a verse or line of poetry.

Reply:
I will rename it to `verse`, as I'm open to your valuable suggestions.